### PR TITLE
TLC-55: Moved Allergies and Medicines higher on form, renamed label "…

### PR DESCRIPTION
**Pull request by ouiliam**
…Other Illnesses and Injuries"

	modified:   configuration/concepts/concepts.csv
	modified:   configuration/concepts/consultation_form.csv

### CODE REVIEW

File: configuration/concepts/consultation_form.csv [hunk 1]
```diff
@@ -14,4 +14,4 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:k
...
\ No newline at end of file
+,,,Cambodia_Consultation Form,Cambodia_Consultation Form,Consultation,ការពិនិត្យរោគ,Consultation,,ConvSet,N/A,,,Cambodia_Chief Complaint; Cambodia_History of Present Illness; Cambodia_Old Case;Cambodia_Past Medical History; Cambodia_Past Cancer Details; Cambodia_Past Medical History Details; Cambodia_Other Past Medical History; Cambodia_Past Surgical History; Cambodia_Allergy History; Cambodia_Medications Set; Cambodia_Social History; Cambodia_Newborn Immunizations Set; Cambodia_Child Immunizations Set; Cambodia_Pregnant Women Immunizations Set; Cambodia_General Set; Cambodia_Skin Set;Cambodia_HEENT Set;Cambodia_Head and Neck Set;Cambodia_Ears Set;Cambodia_Eyes Set;Cambodia_Other HEENT Signs Set;Cambodia_Chest and Lungs Set;Cambodia_Cardiovascular Set;Cambodia_Endocrine Set;Cambodia_Abdomen Set;Cambodia_Genitourinary Set;Cambodia_Flank and Back Set;Cambodia_Musculoskeletal and Neurological Set;Cambodia_Other Physical Signs Set;,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/consultation_form.csv:17]: Why is this line changed that much? Did you add 5 or 6 concept answers/members to it?
Though that is working just fine so...

Comment (ouiliam) [reply configuration/concepts/consultation_form.csv:17]: Yes, before I had the whole past history form in there, but replaced it with it's members.

Comment (rbuisson) [reply configuration/concepts/consultation_form.csv:17]: ok.
